tensorflow/python/data/experimental/ops/readers.py

```diff
@@ -420,6 +420,28 @@ def make_csv_dataset_v2(
 
   Raises:
     ValueError: If any of the arguments is malformed.
+  Usage Example:
+  
+  Using IRIS dataset to show how to convert .csv file into a dataset.
+  
+  ```python
+  >>> train_dataset_url = "https://storage.googleapis.com/download.tensorflow.org/data/iris_training.csv"
+  >>> train_dataset_fp = tf.keras.utils.get_file(fname=os.path.basename(train_dataset_url), origin=train_dataset_url)
+  >>>
+  >>> column_names = ['sepal_length', 'sepal_width', 'petal_length', 'petal_width', 'species']
+  >>> feature_names = column_names[:-1]
+  >>> label_name = column_names[-1]
+  >>> 
+  >>> batch_size = 32
+  >>> train_dataset = tf.data.experimental.make_csv_dataset(
+  ... train_dataset_fp,
+  ... batch_size,
+  ... column_names=column_names,
+  ... label_name=label_name,
+  ... num_epochs=1)
+  >>> features, labels = next(iter(train_dataset))
```

Comment:
Can you rename it to `features_batch`, `labels_batch`? The name `features` suggests that it is the complete dataset.